library/blocks/in-world-recipes/src/main/java/org/quiltmc/qsl/inworldrecipes/api/InWorldRecipeRegistries.java

```diff
@@ -0,0 +1,31 @@
+package org.quiltmc.qsl.inworldrecipes.api;
+
+import com.mojang.serialization.Lifecycle;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.util.registry.RegistryKey;
+import net.minecraft.util.registry.SimpleRegistry;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * <b>NOTE:</b> You need to handle action sound playback yourself.<br>
+ * <b>NOTE2:</b> You <em>can</em> override vanilla recipes with this.
+ */
+public final class InWorldRecipeRegistries {
+	private InWorldRecipeRegistries() { throw new AssertionError(); }
+
+	public static final Registry<InWorldRecipe> SWORD = create("sword");
```

Comment:
These seem to be based on tool and not action, such as tilling, pathing and debarking. Is there a reason for this name scheme? Or is it more abstract than those specific actions.

Reply:
It's more abstract than those specific actions, I feel.

Like, right clicking with a sword could mean carving, stabbing, slicing, etc... the same ambiguity could be applied to the other tool types.